CHANGELOG.md

```diff
@@ -1,14 +1,17 @@
-vX.Y.Z - TBD
+v2.1.0 - 01 Dec 2020
```

Comment:
```suggestion
v2.1.0 - 01 December 2020
```

Absolute annoying nit-pick. But we've never abbreviated the month before, so for consistency, I'd suggest we don't break the pattern now 😄